docs/stencil-docs/installing-stencil-cli/installing-stencil.md

```diff
@@ -97,7 +99,7 @@ If you're a pro at installing and configuring Python and Node.js environments on
 **Required Dependencies:**
 * [Git](https://git-scm.com/downloads) - required to run npm install
 * [Python 2.7.x](https://www.python.org/downloads/) - required to build some dependencies
-* [Node.js 10.17 and npm](https://nodejs.org/en/download/releases/) - later versions not currently supported on Windows
+* [Node.js 12 and npm](https://nodejs.org/en/download/releases/) - later versions not currently supported on Windows

```

Comment:
Need a fact check here. Are versions later than Node v10.17 still not supported on Windows?

Reply:
I tried it on my personal Windows machine and v10.17 still works as well as v12. I can keep it as it is.